src/uct/sm/scopy/knem/knem_ep.c

```diff
@@ -37,34 +38,54 @@ UCS_CLASS_DEFINE_DELETE_FUNC(uct_knem_ep_t, uct_ep_t);
         return UCS_OK;                                  \
     }
 
+
+static UCS_F_ALWAYS_INLINE
+void uct_knem_iov_set_length(struct knem_cmd_param_iovec *iov, size_t length)
+{
+    iov->len = length;
+}
+
+static UCS_F_ALWAYS_INLINE
+void uct_knem_iov_set_buffer(struct knem_cmd_param_iovec *iov, void *buffer)
+{
+    iov->base = (uintptr_t)buffer;
+}
+
+static UCS_F_ALWAYS_INLINE
+size_t uct_iov_to_knem_io_vec(struct knem_cmd_param_iovec *io_vec, size_t *io_vec_cnt_p,
```

Comment:
imo no need in this func: it is just a wrapper which is called once

Reply:
done